lib/shim/message-shim.js

```diff
@@ -589,43 +516,13 @@ function recordSubscribedConsume(nodule, properties, spec) {
     promise: false
   })
 
-  // Make sure our spec has what we need.
```

Comment:
Maybe I should bring these back. I got rid of them because I figured it was better to just error out completely if you gave this function the wrong arguments than to silently proceed and write something to debug output.

It's just a complete error of custom instrumentation to supply the wrong arguments here, so it's not exactly like getting rid of these two checks breaks user code.